### ISSUE TITLE
If Features are further down compared to the Set of Steps, have to scroll up to find the Steps

### ISSUE THREAD

**Issue by Tafhim** [orionhealth/XBDD#329]
If you have a lot of Features in your menu and the feature that you want to work on is too far down, it looks confusing as if Feature does not have any Set of Steps.

Probably can show a count of Steps in each feature
Or cause a scroll to top when there are in fact Steps in a feature
Or have the Steps block fixed / floating in the view constantly?

[https://imgur.com/ZJNQzxc](A blurred screenshot)

**Comment by ryan-rushton** [comment]
Yeah I brought this up the other day, it would be good if we scrolled in each lane rather then the whole page. Then you wouldn't lose the navbar either
